scripts/list-languages-in-enterprise.js

```diff
@@ -0,0 +1,100 @@
+const { listAllRepositoriesInOrganization } = require("../util/github");
+
+const getTotalCodeSizeInBytes = (obj) => Object.values(obj).reduce((accumulator, value) => accumulator + value, 0);
+
+module.exports = {
+    description: "list language usage within an enterprise",
+    options: {
+        enterprise: {
+            alias: "e",
+            demandOption: true,
+            describe: "the GitHub enterprise to search for language usage",
+            type: "string",
+        },
+        percent: {
+            alias: "p",
+            default: 25,
+            describe: "what percentage of code is needed to qualify as a \"used\" language",
+            type: "number",
+        },
+    },
+    action: async (octokit, graphql, argv) => {
+        const organdrepo = [];
```

Comment:
```suggestion
        const orgAndRepo = [];
```

Might as well camel case this like the other variables